### PR TITLE
Drop support for Node 12.x, use the latest version of dependencies

### PR DESCRIPTION
**Pull request by kizule**
Based on https://github.com/wikimedia/eslint-config-wikimedia/pull/483 and others.

### CODE REVIEW

**Comment by jdforrester** [comment]
Please split this into different PRs for different concerns.

**Comment by kizule** [comment]
Okay, I've removed changing version part from this pull request, thanks.

File: package.json [hunk 1]
```diff
@@ -28,21 +28,21 @@
 		"src"
 	],
 	"dependencies": {
-		"eslint-plugin-vue": "^8.7.1",
+		"eslint-plugin-vue": "^9.9.0",
 		"upath": "^2.0.1"
 	},
 	"peerDependencies": {
-		"eslint": ">=5.0.0"
```
Comment (jdforrester) [inline package.json:35]: We are not dropping this?

**Comment by jdforrester** [comment]
As I said, please submit a change for each dependency you wish to propose.

**Comment by kizule** [comment]
> As I said, please submit a change for each dependency you wish to propose.

You already started to do it, thanks! :)

**Pull request closed without merging**